tests/Builder/SourceFileParametersTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Website\Tests\Builder;
+
+use Doctrine\Website\Builder\SourceFileParameters;
+use PHPUnit\Framework\TestCase;
+
+class SourceFileParametersTest extends TestCase
```

Comment:
Please add a test where an unknown parameter is called with `getParameter()` and we expect `null`. This is uncovered code.